app/build.gradle

```diff
@@ -71,27 +72,33 @@ android {
 }
 
 dependencies {
-    api "androidx.appcompat:appcompat:$rootProject.ext.androidXVersion"
-    api "androidx.recyclerview:recyclerview:$rootProject.ext.androidXVersion"
-    annotationProcessor "androidx.annotation:annotation:$rootProject.ext.androidXVersion"
-
-    implementation('com.afollestad.material-dialogs:core:0.9.6.0')
+    implementation "androidx.appcompat:appcompat:1.1.0"
+    implementation "androidx.recyclerview:recyclerview:1.0.0"
+    implementation "com.google.android.material:material:$materialVersion"
 
     implementation('com.tbruyelle.rxpermissions2:rxpermissions:0.9.5') {
         exclude module: 'rxjava'
     }
-    implementation 'io.reactivex.rxjava2:rxjava:2.2.2'
-    implementation 'io.reactivex.rxjava2:rxandroid:2.1.0'
-    implementation 'com.github.akarnokd:rxjava2-interop:0.13.2'
-    implementation('com.github.piasy:RxQrCode:1.3.0')
-    implementation 'com.squareup.leakcanary:leakcanary-android:1.6.2'
+    implementation('com.github.akarnokd:rxjava2-interop:0.13.7') {
+        exclude module: 'rxjava'
+    }
+
+    implementation 'io.reactivex.rxjava2:rxjava:2.2.13'
+    implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
+    implementation 'com.github.piasy:RxQrCode:1.3.0'
+
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:2.0-beta-3'
 
     implementation project(':BigImageViewer')
     implementation project(':FrescoImageLoader')
     implementation project(':GlideImageLoader')
     implementation project(':FrescoImageViewFactory')
     implementation project(':GlideImageViewFactory')
-    annotationProcessor "com.github.bumptech.glide:compiler:$rootProject.ext.glideVersion"
-
     implementation project(':ProgressPieIndicator')
+
+    implementation "com.davemorrissey.labs:subsampling-scale-image-view:$rootProject.ext.ssivVersion"
```

Comment:
That's why I use `api` in library module, by using `api`, library users don't have to declare these dependencies.

Reply:
It's not wrong to use api, it's just not generally a good practice, because it's useful if you have a dep to the core module that you know it's going to be used in all the modules, but in general you want to keep the deps tight so you know what's using what.
It's the same reason why you would not use public visibility on every function in your code, it'll work the same, but you want to keep a tight seal on what is internal or not.
You can see how it's used here: https://github.com/chrisbanes/tivi
Just a couple of modules are using SSIV, one being the sample, so I thought of moving it to implementation.
With this said I can revert it, if you want.

Reply:
If some guy decide to use BIV, it doesn't make any sense that he still need declare dependency of SSIV. The same as fresco loader and fresco, etc.

So please revert this change, thanks.